versions/1/changelog.yml

```diff
@@ -20,4 +20,7 @@
     link: https://github.com/elastic/package-spec/pull/79
   - description: Allow `condition` under inputs.
     type: enhancement
-    link: https://github.com/elastic/package-spec/pull/87
\ No newline at end of file
+    link: https://github.com/elastic/package-spec/pull/87
```

Comment:
I think that this PR link is not valid: https://github.com/elastic/package-spec/pull/91